.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Please remove this change. 

Btw, I usually add things like that to my `~/.gitignore` and they're getting ignored in all git directories.